code/addons/outline/src/withOutline.ts

```diff
@@ -7,7 +7,7 @@ import outlineCSS from './outlineCSS';
 
 export const withOutline = (StoryFn: StoryFunction<Renderer>, context: StoryContext<Renderer>) => {
   const { globals } = context;
-  const isActive = globals[PARAM_KEY] === true;
+  const isActive = globals[PARAM_KEY] || false;
```

Comment:
This will make any string enable the outline, not just `'true'`.
So `/?path=/story/xxxx&globals=outline:false` would actually turn on outline.
I think a stricter check would also solve the issue.
```suggestion
  const isActive = [true, 'true'].includes(globals[PARAM_KEY]);
```

Reply:
Done.